src/ignitus-Shared/ignitus-DesignSystem/ignitus-Molecules/ignitus-ToggleButtons/Components/index.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import * as B from '../../../ignitus-Atoms/buttons';
+import { ToggleButton1, ToggleButton2, StyledIcon } from '../styles';
+import * as C from '../../../..';
+import { InputTypes1, InputTypes2 } from '../types';
+
+const ButtonLeft = styled(B.Button)`
+  font-weight: 600;
+  margin: 0;
+  border: 0;
+  float: left;
+  border-radius: 20px 0 0 20px;
+`;
+
+const ButtonRight = styled(B.Button)`
+  font-weight: 600;
+  margin: 0;
+  border: 0;
+  float: left;
+  border-radius: 0 20px 20px 0;
+`;
+
+export const DefaultToggleButton1 = ({
+  leftLabel,
+  rightLabel,
+}: InputTypes1) => (
```

Comment:
```suggestion
export const DefaultToggleButton1: React.FC<InputTypes1> = ({
  leftLabel,
  rightLabel,
}) => (
```

Reply:
Which category shall I assign the button, when it is selected?